infra/build/functions/base_images.py

```diff
@@ -103,3 +138,12 @@ def base_builder(event, context):
   images = [tag_prefix + base_image for base_image in BASE_IMAGES]
 
   run_build(steps, images)
+
+  introspector_steps = _get_introspector_base_images_steps(
+      INTROSPECTOR_BASE_IMAGES, tag_prefix)
+  intro_images = [
```

Comment:
nit: s/intro/introspector/. Always better to be more explicit (and consistent) here.

Reply:
Done.